client/src/components/SupportTools/support-api.tsx

```diff
@@ -24,6 +24,8 @@ export interface IElectionBase {
     | 'HYBRID'
   online: boolean
   deletedAt: string | null
+  createdAt: string
+  currentRound: IRound | null
```

Comment:
I think `IElectionBase` is used in a few different types, but you didn't add these fields to all of them (e.g. `IJurisdiction`), so I think this will end up being a bit inaccurate. Instead, I think you should just add these fields to the types that represent the response from the appropriate API endpoints